storage/README.md

```diff
@@ -1,9 +1,13 @@
 # Azure Storage SDK for Go (Preview)
 
 :exclamation: IMPORTANT: This package is in maintenance only and will be deprecated in the
-future. Consider using the new package for blobs currently in preview at
-[github.com/Azure/azure-storage-blob-go](https://github.com/Azure/azure-storage-blob-go).
-New Table, Queue and File packages are also in development.
+future. Please use one of the following packages instead.
```

Comment:
since we have all 3 of them now, dont we want to say they are deprecated and will be removed in the future ?

Reply:
and also actually deprecate it. and I saw you are doing the PR directly to master, dont we want to put it in latest ?

---
In reply to: [195187726](https://github.com/Azure/azure-sdk-for-go/pull/1911#discussion_r195187726) [](ancestors = 195187726)

Reply:
maybe, but I'm not yet sure how we'll handle tables so didn't change this yet. will open an issue

Reply:
https://github.com/Azure/azure-sdk-for-go/issues/2037